build

```
@@ -131,12 +131,12 @@ def compile_docker(config, target, src_dir, tgt_dir, debug=False):
     if os.path.exists(os.path.abspath(script)):
         dshell('/tgt/app', '/pkg/%s' % script)
 
-def package_docker(config, target, src_dir, iteration, clean=False):
+def package_docker(config, target, src_dir, iteration, clean=False, debug=False):
     tgt_dir = os.path.join('targets', target)
     if clean:
         shell('rm -fr %s' % tgt_dir)
 
-    compile_docker(config, target, src_dir, tgt_dir)
+    compile_docker(config, target, src_dir, tgt_dir, debug=debug)
 
     pkg_ver, pkg_iter = get_version(src_dir, iteration)
```

```suggestion
    pkg_ver, pkg_iter = get_version(src_dir, iteration)
    if debug:
        pkg_iter += ".debug"
```
Can you check if this is used, it generates a different package name? In that case, I might be okay to merge it with this implemented.

---

Looking at the code that looks like this probably should work, testing a build right now.

Do you know if there's a reason `--clean` both deletes the target dir before and after compiling? I wanted a clean build from the last build I ran, but was expecting to look at the intermediate files and didn't realize it nuked the directory after the build. I ran it with the wrong arguments so I needed to re-run anyways, but can I remove the `rm -fr` post build and just leave the pre-build clean up?

---

This built `wkhtmltox_0.12.7-0.20220705.dev.debug.buster_amd64.deb` as expected.